php-practice/section1.php

```diff
@@ -186,6 +230,16 @@
 // =======================================================================
 
 
+$myAssocArray = [
+	'id' => 5, 
+	'color' => 'white',
+	'fallingToItsDeath' => false,
+	'rhinosEndangered' => true
+	];
+
+	unset($myAssocArray[1]);
```

Comment:
@paulstatezny it didn't work because the Associative Array's indexes are not numbers but strings. So I can't put unset($myAssocArray[1]) instead I have to do unset($myAssocArray['color']) so it knows to unset that string index.